src/js/media-chrome-range.js

```diff
@@ -121,9 +127,8 @@ template.innerHTML = `
     input[type=range]:disabled::-webkit-slider-runnable-track {
       background-color: #777;
     }
-
   </style>
-  <input id="range" type="range" min="0" max="1000" step="1" value="0">
+  <input id="range" type="range" part="range" min="0" max="1000" step="1" value="0">
```

Comment:
We probably don't want to do this right now. @cjpillsbury believes we'll need to switch to a non-native range implementation at some point, and this would make that a breaking change compared to exposing css vars we can translate between implementations.

Reply:
makes sense, I'll remove this part attribute.